Tools/wasm/emscripten/node_pre.js

```diff
@@ -1,9 +1,15 @@
 // If process is undefined, we're not running in the node runtime let it go I
 // guess?
 if (typeof process !== "undefined") {
-    const nodeVersion = Number(process.versions.node.split('.',1)[0]);
+    const nodeVersion = Number(process.versions.node.split(".", 1)[0]);
     if (nodeVersion < 18) {
-        process.stderr.write(`Node version must be >= 18, got version ${process.version}\n`);
+        process.stderr.write(
+            `Node version must be >= 18, got version ${process.version}\n`,
+        );
         process.exit(1);
     }
+    Module.preRun = () => {
+        FS.mkdirTree("/lib/");
+        FS.mount(NODEFS, { root: __dirname + "/lib/" }, "/lib/");
+    };
```

Comment:
Is there a reason this wasn't needed previously?

Reply:
With `-sNODERAWFS`, the emulated file system is set up to match the native file system as closely as possible. So this is automatic. Now that we are not doing that, we need to mount the standard library so that the Python interpreter will start up. Ideally we'd be able to select whether or not to use `-sNODERAWFS` at runtime, but it's a node only option.

I think in a followup I'll update it to mount most of the native file system directories. But I want to change the way a few more things work first (and there are some bugs that make this a bit weirder than it could be https://github.com/emscripten-core/emscripten/issues/22924).